src/display/api.js

```diff
@@ -717,6 +717,8 @@ var PDFDocumentProxy = (function PDFDocumentProxyClosure() {
  * @property {Object} canvasFactory - (optional) The factory that will be used
  *                    when creating canvases. The default value is
  *                    {DOMCanvasFactory}.
+ * @property {string} backgroundColor - (optional) Background color to use for
+ *                    the canvas. The default value is 'rgb(255,255,255)'.
```

Comment:
nit: can you make a note that the color is specified in the CSS format? Actually it can be string, CanvasGradient, or CanvasPattern -- someone creative might apply paper texture as a background, no?